Modules/_sha3/README.txt

```diff
@@ -1,11 +1,5 @@
-Keccak Code Package
-===================
-
-The files in kcp are taken from the Keccak Code Package. They have been
-slightly to be C89 compatible. The architecture specific header file
-KeccakP-1600-SnP.h ha been renamed to KeccakP-1600-SnP-opt32.h or
-KeccakP-1600-SnP-opt64.h.
-
-The 64bit files were generated with generic64lc/libkeccak.a.pack target, the
-32bit files with generic32lc/libkeccak.a.pack.
+tiny_sha3
+=========
 
+https://github.com/mjosaarinen/tiny_sha3
```

Comment:
include its LICENSE file.

Reply:
The license was already in the tiny_sha3 sub directory. I moved code and license next to sha3module.c in the latest commit.